specifyweb/frontend/js_src/lib/components/Interactions/LoanReturn.tsx

```diff
@@ -104,6 +104,36 @@ function PreparationReturn({
     ({ resolve, unresolved }) => resolve < unresolved
   );
 
+  const [bulkReturn, setBulkReturn] = React.useState(0);
+  const [bulkResolve, setBulkResolve] = React.useState(0);
+  const quantities = preparations.map(
+    (preparation) => preparation.get('quantity') ?? 0
+  );
+  const maxPrep = Math.max(...quantities);
```

Comment:
can quantities.length ever be 0 here? if so, use f.max instead of Math.max as then TypeScript will force you to handle this edge case